fulp_modules/features/nanites/code/programs/utility.dm

```diff
@@ -176,9 +176,10 @@
 	var/spread_cooldown = 0
 
 /datum/nanite_program/spreading/active_effect()
-	if(spread_cooldown < world.time)
+	if(spread_cooldown > world.time)
```

Comment:
Actually, do you think you can replace this with ``COOLDOWN_DECLARE``, ``COOLDOWN_FINISHED``, ``COOLDOWN_START``, like I did with the nanite chamber, nanite programmer, and other programs? It would make it more readable and bring this in line to other programs.